examples/pytorch/question-answering/run_seq2seq_qa.py

```diff
@@ -476,15 +483,18 @@ def preprocess_validation_function(examples):
             max_length=max_seq_length,
             padding=padding,
             truncation=True,
-            return_overflowing_tokens=True,
             return_offsets_mapping=True,
+            # return_overflowing_tokens=True,
```

Comment:
This shouldn't be commented out.

Reply:
I think I'm going to keep it because this problem https://github.com/huggingface/transformers/issues/15398

Reply:
This is not a fix of the issue, just a downgrade of the example.

Reply:
This is an inherent problem, so I will take it back.